### PR TITLE
Defloat Etch

### PR DESCRIPTION
**Pull request by wkaluza**

### CODE REVIEW

File: libs/vm-modules/include/vm_modules/math/bignumber.hpp [hunk 1]
```diff
@@ -65,14 +65,8 @@ class UInt256Wrapper : public fetch::vm::Object
   static fetch::vm::Ptr<UInt256Wrapper> ConstructorFromBytes(
       fetch::vm::VM *vm, fetch::vm::TypeId type_id, fetch::vm::Ptr<ByteArrayWrapper> const &ba);
 
-  double ToFloat64() const;
-
   int32_t ToInt32() const;
 
-  double LogValue() const;
-
-  bool LessThan(fetch::vm::Ptr<UInt256Wrapper> const &other) const;
```
Comment (wkaluza) [inline libs/vm-modules/include/vm_modules/math/bignumber.hpp:74]: `LessThan` was unused; in any case, the VM operator binding is `IsLessThan`.

File: libs/vm-modules/tests/unit/fixed_point.cpp [hunk 2]
```diff
@@ -217,24 +217,25 @@ TEST_F(FixedPointTest, multidigit_integral_fixed_point64)
...
-      var val64_1 = 123.456fp64;
-      assert(abs(val128_3 - toFixed128(val32_1)) < toFixed128(0.0003));
-      assert(abs(val128_3 - toFixed128(val64_1)) < toFixed128(0.00000012));
-    endfunction
-  )";
-
-  EXPECT_TRUE(toolkit.Compile(TEXT));
-  EXPECT_TRUE(toolkit.Run());
-}
+//???
```
Comment (markos) [inline libs/vm-modules/tests/unit/fixed_point.cpp]: why is this commented out? it's not using float?

Comment (wkaluza) [reply libs/vm-modules/tests/unit/fixed_point.cpp]: Uncommented now; I was initially commenting everything out to save time

**Review by ejfitzgerald — DISMISSED** [review]
Looks good from my side

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**